### ISSUE TITLE
Addon tab not shown in Storybook 7.0.0-alpha.46

### ISSUE THREAD

**Issue by interphased** [pocka/storybook-addon-designs#172]
### Affected design types

- [X] figma
- [X] figspec
- [X] link
- [X] image
- [X] iframe

### Describe the bug

I'm still having [this issue](https://github.com/pocka/storybook-addon-designs/issues/171) in `Storybook 7.0.0-alpha.46` and `storybook-addon-designs 7.0.0-beta.0`.

```
index.js:10 Uncaught TypeError: addons_default.register is not a function
    at register (index.js:10:12)
    at register.js:3:1
```

If I update the addons import in `src/register/index.tsx` like so:

```
- import addons, { types } from "@storybook/addons";
+ import { addons, types } from "@storybook/addons";
```

The problem goes away, and now the tab appears in Storybook, but then I get other issues:

<img width="870" alt="image" src="https://user-images.githubusercontent.com/1676931/199820285-98229761-ec93-4707-904e-005a8239a1d7.png">

Strangely, if I find the iframe src and view it in a new tab it shows up. Maybe that's a separate issue with Figma and localhost permissions.

### How to reproduce the bug?

See [this issue](https://github.com/pocka/storybook-addon-designs/issues/171).

Note that I'm using `@storybook/react-vite` framework:

```
// main.cjs

module.exports = {
  stories: ['../src/**/*.stories.mdx', '../src/**/*.stories.@(js|jsx|ts|tsx)'],
  addons: [
    ...
    'storybook-addon-designs',
  ],
  framework: {
    name: '@storybook/react-vite',
    options: {},
  },
  core: {},
  staticDirs: ['../public'],
  async viteFinal(config) {
    return mergeConfig(config, {
      resolve: {
        alias: {
          '@': path.resolve(__dirname, '../src'),
        },
      },
    });
  },
};
```

### Expected behaviour

To show the design tab.

### Environment

- [X] Chrome / Chromium / Chromium based browsers
- [X] Firefox
- [ ] macOS Safari
- [ ] iOS Safari
- [ ] Samsung Internet
- [ ] Other

### Affected versions

7.0.0-beta.0

### Storybook versions

7.0.0-alpha.46

**Comment by pocka** [comment]
> If I update the addons import in src/register/index.tsx like so:

You're right. That file didn't get the update in #160... This should be an easy fix.

> Strangely, if I find the iframe src and view it in a new tab it shows up. Maybe that's a separate issue with Figma and localhost permissions.

Yes, the addon is working okay when logs from `mobile_viewer.min.js.br` appears in a browser console.
I'm not sure what the WS warning (the first one) is, but you can safely ignore the source map warnings (the last two).

**Comment by interphased** [comment]
Ok great. I've [added a PR](https://github.com/pocka/storybook-addon-designs/pull/173).

**Comment by pocka** [comment]
Fixed by #173.

**Comment by mnik01** [comment]
facing this issue in version     "storybook-addon-designs": "7.0.0-alpha.3",
